### PR TITLE
Add basic highlighting functionality

### PR DESCRIPTION
**Pull request by AwesomestCode**
Add highlighting to make it easier to break/place blocks.

### CODE REVIEW

**Comment by u9g** [comment]
does this color the whole block, or just the outline?

**Comment by Karang** [comment]
> does this color the whole block, or just the outline?

this is the full block, but the ourline can be made as well

example: https://github.com/PrismarineJS/prismarine-viewer/blob/master/viewer/lib/primitives.js#L29

File: index.js [hunk 1]
```diff
@@ -14,10 +14,45 @@ const { WorldView, Viewer } = require('prismarine-viewer/viewer')
...
+
+  points.push(new THREE.Vector3(x + size + offset, y - offset, z))
+  points.push(new THREE.Vector3(x + size + offset, y + size + offset, z)) // connector
+  points.push(new THREE.Vector3(x + size + offset, y - offset, z))
+
+  points.push(new THREE.Vector3(x + size + offset, y - offset, z + size + offset))
+
+  const geometry = new THREE.BufferGeometry().setFromPoints(points)
+
+  highlightCube.geometry = geometry
```
Comment (Karang) [inline index.js:53]: You are recreating the geometry each time you move the cube: it is not efficient.
Instead move it using position.set() like you did before.

**Comment by AwesomestCode** [comment]
fixed

**Comment by AwesomestCode** [comment]
For some reason it doesn't show half of the time. I have no idea why. I'm leaving it here since I don't know how to fix it, I'll start working on another issue later.

**Comment by rom1504** [comment]
I confirm it worked 2 commits ago, not sure why the editing the geometry makes it unreliable?

**Comment by rom1504** [comment]
ok it now works but:
* it's changing the highlight every time even if the block didn't change (should be avoided)
* there is no distance limit (I think in mc the limit is 6 block)
* the highlight should be hidden if no block is pointed

want to try doing that @AwesomestCode ?

**Comment by rom1504** [comment]
and this:
> extremeheat : I think it would be easier to just draw the outline for the whole block and just not worry about blocks with weird bbs for now. just creating the geometry once and translating it should work

**Comment by AwesomestCode** [comment]
Sure.

**Comment by AwesomestCode** [comment]
Superseded by #93.

**Pull request closed without merging**